### PR TITLE
isTemplate refactor, moving from UI to sdk

### PR DESCRIPTION
**Pull request by bthaile**
refactor to move isTemplate processing to sdk, first step to getMarkets isTemplate filter.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🏄‍♂️

 This link will always point to this PR's latest changes: https://peek.run/jvghvxr

File: packages/augur-tools/src/templates-template.ts [hunk 1]
```diff
@@ -153,13 +153,65 @@ export enum TemplateInputType {
   USER_DESCRIPTION_DROPDOWN_OUTCOME = 'USER_DESCRIPTION_DROPDOWN_OUTCOME',
 }
 
+export interface ExtraInfoTemplateInput {
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/templates-template.ts:156]: This file is heavily duplicated in `packages/augur-artifacts/src/templates.ts`.

Comment (bthaile) [reply packages/augur-tools/src/templates-template.ts:156]: it's the templates template file. it's used to generate the file in artifacts.

Comment (robert-e-davidson3) [reply packages/augur-tools/src/templates-template.ts:156]: ohhhh

**Review by robert-e-davidson3 — APPROVED** [review]
lgtm

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 💫
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/jvghvxr**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Pull request merged**